tee-worker/ts-tests/integration-tests/resuming_worker.test.ts

```diff
@@ -391,6 +391,9 @@ describe('Resume worker', function () {
     let worker1State: WorkerState | undefined = undefined;
 
     step('Two workers & resume worker1', async function () {
```

Comment:
Why dont' you just use `xstep` ? 🙃

Reply:
Because I didn't know it was a thing :sweat_smile: Mocha != Jest apparently :upside_down_face: